edit/namespace.vue

```diff
@@ -31,14 +33,24 @@ export default {
     if ( this.originalValue?.metadata?.name ) {
       originalQuotaId = `${ this.originalValue.metadata.name }/default-quota`;
     }
+    let projectName = this.value?.metadata?.labels?.[PROJECT] || this.$route.query[PROJECT_ID];
+    const projects = this.$store.getters['management/all'](MANAGEMENT.PROJECT);
 
-    const project = this.value?.metadata?.labels?.[PROJECT] || this.$route.query[PROJECT_ID];
+    const project = projects.find(p => p.id.includes(projectName));
 
-    return {
-      originalQuotaId,
-      project,
-      containerResourceLimits: this.value.annotations[CONTAINER_DEFAULT_RESOURCE_LIMIT] || this.getDefaultContainerResourceLimits(project)
-    };
+    // namespaces' project label remains when a project has been deleted: verify this project still exists
+    if (!project && this.value?.metadata?.labels?.[PROJECT]) {
+      delete this.value.metadata.labels[PROJECT];
+      projectName = null;
+    }
+
+    if (this.mode === 'edit') {
```

Comment:
This breaks a few things when in view mode

Reply:
This if block...doesn't need to be here at all. I don't know how that got there. Removed.